data.sql

```diff
@@ -1,6 +1,37 @@
 insert into animals(name,date_of_birth,weight_kg,neutered, escape_attempts) 
 values 
-    ('Agumon', '2020-02-03', 10.23, true, 0),
-    ('Gabumon', '2018-11-15', 8, true, 2),
-    ('Devimon', '2017-05-12', 11, true, 5),
-    ('Pikachu', '2021-01-07', 15.04, false, 1);
\ No newline at end of file
+
+  ('Charmander', '2020-02-08', -11, false, 0),
+  ('Plantmon', '2021-11-15', -5.7, true, 2),
+  ('Squirtle', '1993-04-02', -12.13, false, 3),
+  ('Angemon', '2005-06-12', -45, true, 1),
+  ('Boarmon', '2005-06-07', 20.4, true, 7),
+  ('Blossom', '1998-10-13', 17, true, 3),
+  ('Ditto', '2022-05-14', 22, true, 4);
+
+insert into owners(full_name, age)
```

Comment:
* I get this error when I run these queries reasons because you do not have a owners and species table, Kindly fix this and run it to be sure it gives you the right results, also replace the screenshots on your PR as the queries you have are different from what you have on your screenshots 
![image](https://user-images.githubusercontent.com/52497246/222197345-17f8f22a-c24a-465c-ad2e-326c360c2116.png)